src/utils/useNavigation.ts

```diff
@@ -108,7 +111,11 @@ const useNavigation = () => {
       axios
         .get(`${getChromeStaticPathname('navigation')}/${currentNamespace}-navigation.json`)
         // fallback static CSC for EE env
-        .catch(() => axios.get<BundleNavigation>(`${isBeta() ? '/beta' : ''}/config/chrome/${currentNamespace}-navigation.json?ts=${Date.now()}`))
+        .catch(() => {
+          const LOCAL_PREVIEW = localStorage.getItem('chrome:local-preview') === 'true';
```

Comment:
@Hyperkid123 more of a nit - but I would suggest we move this to a helper function so we avoid any mis-spellings or incorrect local storage checks. I'd be fine exporting this temporarily from `state/atoms/releaseAtom` as a sort of temporary helper.

Reply:
@Hyperkid123 thoughts ☝️ ?